src/pages/home/report/ReportActionsView.tsx

```diff
@@ -299,76 +309,116 @@ function ReportActionsView({
-        // and there are fewer than 23 items, indicating we've reached the oldest message.
-        const isLoadingOlderReportsFirstNeeded = checkIfContentSmallerThanList() && reportActions.length > 23;
-
-        Log.info(
-            `[ReportActionsView] loadNewerChats ${JSON.stringify({
-                isOffline: network.isOffline,
-                isLoadingOlderReportActions,
-                isLoadingInitialReportActions,
-                newestReportAction: newestReportAction.pendingAction,
-                firstReportActionID: newestReportAction?.reportActionID,
-                isLoadingOlderReportsFirstNeeded,
-            })}`,
-        );
+                // Get newer actions based on the newest reportAction for the transaction thread report
+                const oldestActionTransactionThreadReport = reportActionIDMap.findLast((item) => item.reportID === transactionThreadReport.reportID);
+                Report.getNewerActions(oldestActionTransactionThreadReport?.reportID ?? '0', oldestActionTransactionThreadReport?.reportActionID ?? '0');
+            } else {
+                // Retrieve the next REPORT.ACTIONS.LIMIT sized page of comments
+                Report.getOlderActions(reportID, oldestReportAction.reportActionID);
+            }
+        },
+        [
+            network.isOffline,
+            isLoadingOlderReportActions,
+            isLoadingInitialReportActions,
+            oldestReportAction,
+            hasCreatedAction,
+            reportID,
+            reportActionIDMap,
+            transactionThreadReport,
+            hasLoadingOlderReportActionsError,
+        ],
+    );
 
-        if (isLoadingInitialReportActions || isLoadingOlderReportActions || network.isOffline || newestReportAction.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
-            return;
-        }
+    const loadNewerChats = useCallback(
+        (force = false) => {
+            // Determines if loading older reports is necessary when the content is smaller than the list
+            // and there are fewer than 23 items, indicating we've reached the oldest message.
+            const isLoadingOlderReportsFirstNeeded = checkIfContentSmallerThanList() && reportActions.length > 23;
+
+            Log.info(
+                `[ReportActionsView] loadNewerChats ${JSON.stringify({
+                    isOffline: network.isOffline,
+                    isLoadingNewerReportActions,
+                    hasLoadingNewerReportActionsError,
+                    isLoadingInitialReportActions,
+                    newestReportAction: newestReportAction.pendingAction,
+                    firstReportActionID: newestReportAction?.reportActionID,
+                    isLoadingOlderReportsFirstNeeded,
+                })}`,
+            );
+
+            if (
+                !force &&
+                (isLoadingInitialReportActions ||
+                    isLoadingNewerReportActions ||
```

Comment:
This used to check `isLoadingOlderReportActions` which I'm pretty sure is an error so I changed it to `isLoadingNewerReportActions`.